Skuid_Techniques/Navigation/README.MD

```diff
@@ -0,0 +1,167 @@
+
+But if you want truly data driven nav - read on.... 
+
+## Examples: 
+Below are two pages that use the Vertical Navigation Component. 
+
+You will need to ensure the Design System used has Selected State defined so it is noticeable. 
+
+### Vertical navigation 
+- Page API:  V2
+- Data source: None
+- Design system: See Note 
+- Page XML:  [Copy the XML from this page](Veritcal_Nav.xml), or save it as an XML file, and upload it as a new page.  
+
+### Integration of wizard and vertical navigation 
+- Page API:  V2
+- Data source: None
+- Design system: None 
+- Page XML:  [Copy the XML from this page](Wizard_Nav_Integration.xml), or save it as an XML file, and upload it as a new page.  
+
+
+
+
+
+## Option 2:  List or deck component
+
+The same requirements - but now delivered using our data iterator components.  This facilitates building out navigation that is based on Model data -- a truly flexible and infinitely variable navigation. 
+
+### Highlight the users current selection 
+Similar to the Navigation component - the List component has “selected state control”.   It’s configured on the “Click interaction” node.  It does require a selected state to be configured for the List component of the design system.  Look for the “row” properties.  
+
+If the standard selected state control does not provide enough control,  or if you are using a Deck - use style variant conditions on a wrapper or grid in the deck.  Update ui-only fields in your action sequence to highlight the active card.  Make sure you update all rows to “not selected” before you update the current row to “selected”.   
+  
+### Display data in the navigation element 
+Because the iterator components are bound to model data - it is easier to use data.  You can also add HTML to templates in the list. 
+
+You can also use style variant conditions to highlight style of a wrapper in a deck card if certain conditions are met.  Alert to “urgent” attention needed on one navigation element. 
+
+### Disable navigation element based on data.  
+We do not have Enable Condition logic for the interactions associated with Deck Cards or List items.  (Enhancement!)   Instead use Branch Actions to prevent the action for particular scenarios.  You have to define the actions for each navigation item,  but this allows you to have different logic rules for each item.  Your failure branch can post a message about why the page load is not happening. 
+
+### Collapsable Nav 
+Just like the vertical navigation component  - the most straightforward method to have collapsibility is to have 2 components and use conditional rendering to only show one. 
+
+The list component needs to implement column based conditional rendering.  (Enhancement!) 
+
+A deck component will not work so well here.  The collapsed state will want to wrap individual cards instead of shrinking the entire column.   This is a work in progress. 
```

Comment:
```suggestion
A Deck component will not work so well here.  The collapsed state will want to wrap individual cards instead of shrinking the entire column.
```